src/services/membership_api/routes/memberRoute.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
```

Comment:
switch to using **import** statements so as to be in the standard of ES6

Reply:
made changes

Reply:
done

Reply:
done